<p>For readability reasons, the same starting token should be used for each comment line. Rule can be configured to use either <code>!</code> or <code>#</code></p>
<p>This starting token should also be followed with a whitespace.</p>

<h2>Noncompliant Code Example</h2>
With token "#":
<pre>
! My comments...
! My comments...
#My comments...
</pre>

<h2>Compliant Solution</h2>
With token "#":
<pre>
# My comments...
# My comments...
# My comments...
</pre>



