<p>Defining several times the same key leads to unexpected behavior. Thus, you should remove those duplicated keys.</p>
<h2>Noncompliant Code Example</h2>
<pre>
key1=value1
key2=value2
key1=value3
</pre>
<h2>Compliant Solution</h2>
<pre>
key1=value1
key2=value2
key3=value3
</pre>
