<p>Use Domain names instead of IP, usage of IP address in properties file leads to failure in dynamic IP assignment infrastructure.</p>
<h2>Noncompliant Code Example</h2>
<pre>
key1=120.235.200.12
</pre>