<p>
    <code>TODO</code> tags are commonly used to mark places where some more code is required, but which the developer
    wants to implement later. Sometimes the developer will not have the time or will simply forget to get back to that
    tag. This rule is meant to track those tags, and ensure that they do not go unnoticed.
</p>

<h2>Noncompliant Code Example</h2>
<pre>
# TODO: blabla
key1=value1

! TODO
key2=value2
</pre>